### ISSUE TITLE
Comic showing as "Already in Library", but not visible in library. Server throws error when I click on the "Already in Lbrary" link.

### ISSUE THREAD

**Issue by jgkelly** [mylar3/mylar3#764]
**Describe the bug**
Attempting to add the X-Corp comic to my library, the search shows "Already in Library".
![image](https://user-images.githubusercontent.com/1762584/118577281-20c94b00-b758-11eb-8fde-7aa84329db1a.png)

If I return to the main library list and search for X-Corp in my library, the search shows no results:
![image](https://user-images.githubusercontent.com/1762584/118577399-54a47080-b758-11eb-9561-ecf49d82d57a.png)

If I click the "Already in Library" link from the comics search window, the server brings returns a 500 server error: 
![image](https://user-images.githubusercontent.com/1762584/118577339-38a0cf00-b758-11eb-93fb-9681b25df877.png)

I am not sure if this is related, but I've also noticed a "None" comic that results in a 500 error when I attempt to navigate to it:
![image](https://user-images.githubusercontent.com/1762584/118578434-27f15880-b75a-11eb-8216-e6164280c2b6.png)
![image](https://user-images.githubusercontent.com/1762584/118578494-3770a180-b75a-11eb-8019-607632323b71.png)

**CarePackage**
I have the CarePackage.zip, but GitHub will not let me upload it. File size: 510 MB. Is there somewhere else I can upload it to? 

**Environment (please complete the following information):**
 - OS: Raspbian (Debian) 5.10.17-v7l+ #1403 Running on Raspberry Pi 4.
 - Master:[5dce8bf2fe15a2266c372facac14c9675b33ad49]
 - Python verion 3.7.3

**Additional information**
The library was ported from a Windows Server 2012 hosted environment. The port went well and didn't seem to have any problems. 
Please contact me if you need more information from me.

**Comment by evilhero** [comment]
This might have been fixed in the python3-dev branch, but hasn't made its way into the master branch yet. 

Just a few questions:
- do you have a Comic Location  path specified in your configuration that's valid?
- when you ported from Windows to debian - did you happen to update the db paths so that they point to the debain mounts and not old windows mounts (ie. locmove) 
- can you try searching for ```corp``` instead of the full title? There's a weird glitch sometimes that if a value in the dB is invalid, it won't render it during a filter for whatever reason.

What makes up the 500mb in the log file? Is it the db, the config or the log files? If it's the last 2, there was a bug that was fixed recently that caused the config and log to grow excessively, which might be part of what's happening here as well.

**Comment by jgkelly** [comment]
To answer your questions: 
- Yes, I confirmed the comic path (/mnt/library/comics) is valid and accessible to the user running Mylar
- I made sure I updated all paths from the old Windows path to the new paths on the PI, including the DB path (/opt/mylar3)
- Searching for `corp` instead of the full title, then clicking on the "Already in Library" link worked, and brought me to the series' page. 
- I will definitely update Python to 3.8.+. Didn't notice that was so out of date. 

The largest file in the care package is the Mylar.db file at approx. 507 MB. The six log files clocked in at 54 MB altogether. 
I'll also restart Mylar like you suggested. 

Thank you for the help!
